book/website/collaboration/inclusive-events/inclusive-events-resources.md

```diff
@@ -20,5 +21,92 @@
+* Citizen's Advice: [How to run focus groups](https://www.citizensadvice.org.uk/Global/CitizensAdvice/Equalities/How%20to%20run%20focus%20groups%20guide.pdf)
+* [Guidelines for Conducting a Focus Group](https://irep.olemiss.edu/wp-content/uploads/sites/98/2016/05/Trinity_Duke_How_to_Conduct_a_Focus_Group.pdf)
+
+### Community-Specific Accessibility Resources
+
+#### Deaf and Hard of Hearing 
+* [Sign Solutions: How to plan a conference with event interpretation for Deaf people](https://www.signsolutions.uk.com/how-to-plan-a-conference-with-event-interpretation-for-deaf-people/)
+* [Royal National Institute for Deaf People: Make your meetings deaf aware](https://rnid.org.uk/information-and-support/deaf-awareness/make-your-meetings-deaf-aware/)
+* (UK only) [Action Deafness Interpreting: British Sign Language Interpreter Services](https://actiondeafness.org.uk/services/interpreting/)
+* [Working with sign language interpreters for events](https://www.signsolutions.uk.com/working-with-sign-language-interpreters-for-events/)
+
+#### Autism
+* Autistica Advocacy: [A Guide to Accessible Event Planning](https://autisticadvocacy.org/wp-content/uploads/2019/05/Accessible-Event-Planning.pdf)
+* National Autism Society: [A quick, easy guide to holding an autism-friendly event](https://www.autism.org.uk/get-involved/raise-money/do-your-own-thing/how-to-guides/holding-an-event)
+
+### Planning Accessible Events
+* [Creating Accessible Events](https://www.friendlylikeme.com/access-spotlight/creating-accessible-events)
+* [Make Things Accessible: Running Accessible Events](https://www.makethingsaccessible.com/guides/running-accessible-events/)
+* [Disabled Equality Scotland: How to organize an Inclusive and Accessible Event](https://inclusivecommunication.scot/how-to-organise-an-inclusive-and-accessible-event)
+* [University College London's Tips / Checklist for making events accessible](https://www.ucl.ac.uk/equality-diversity-inclusion/equality-areas/disability-equality/tips-checklist-making-events-accessible)
+* [University of British Columbia Accessible and Inclusive Event Planning](https://equity.ubc.ca/resources/accessible-and-inclusive-event-planning/)
+* [University of York: Inclusive and Accessible Events: Good Practice Guidance](https://www.york.ac.uk/media/abouttheuniversity/equality/documents/Inclusive-and-accessible-events-good-practice-guidelines.pdf)
+* Equality and Human Rights Commission: [Engaging with Disabled People An Events Planning Guide](https://www.equalityhumanrights.com/sites/default/files/housing-and-disabled-people-engaging-with-disabled-people-event-planning-guide.pdf)
+* [Independent Living Institute: Holding Meetings](https://www.independentliving.org/docs2/daakit45.html)
+* [Making in-person events accessible](https://communications.admin.ox.ac.uk/guidance-for-planning-and-organising-accessible-events#collapse2055661)
+
+### Planning Inclusive Events
```

Comment:
I haven't checked, but are those also listed in the more general event organisation chapters? If so, would it be worth linking out to the broader resources for inclusive event organisation within TTW (please ignore if i misremember and that doesn't actually exist 🙈 )

Reply:
That actually aren't (but possibly could be!)